src/main/java/ca/corefacility/bioinformatics/irida/service/impl/ProjectServiceImpl.java

```diff
@@ -528,10 +531,14 @@ public void removeSamplesFromProject(Project project, Iterable<Sample> samples)
 	@Override
 	@PreAuthorize("hasPermission(#project, 'canReadProject')")
 	public boolean userHasProjectRole(User user, Project project, ProjectRole projectRole) {
```

Comment:
I cannot find any usages of this function, if you cannot either can we remove this. This doesn't take into account project role through user group join.

Reply:
I should note there are only tests that reference this function, as far as I can see.

Reply:
Not being used anywhere so removed it and it's tests in c427089ae2f1a86e46a20ce76e32f226b2ca3b89